message/infrastructure/sql/subscriber.go

```diff
@@ -200,46 +214,70 @@ func (s *Subscriber) query(
 		}
 	}()
 
-	selectArgs, err := s.config.SchemaAdapter.SelectArgs(topic, s.config.ConsumerGroup)
-	if err != nil {
-		return errors.Wrap(err, "could not get args for the select query")
-	}
-
-	logger.Trace(selectQuery, watermill.LogFields{
-		"args": fmt.Sprintf("%+v", selectArgs),
+	selectQuery, selectQueryArgs := s.config.SchemaAdapter.SelectQuery(
+		topic,
+		s.config.ConsumerGroup,
+		s.config.OffsetsAdapter,
+	)
+	logger.Trace("Querying message", watermill.LogFields{
+		"query":      selectQuery,
+		"query_args": sqlArgsToLog(selectQueryArgs),
 	})
+	row := tx.QueryRowContext(ctx, selectQuery, selectQueryArgs...)
 
-	row := selectStmt.QueryRowContext(ctx, selectArgs...)
-
-	var offset int
-	var msg *message.Message
-	offset, msg, err = s.config.SchemaAdapter.UnmarshalMessage(row)
+	offset, msg, err := s.config.SchemaAdapter.UnmarshalMessage(row)
 	if errors.Cause(err) == sql.ErrNoRows {
 		// wait until polling for the next message
+		logger.Debug("No more messages, waiting until next query", watermill.LogFields{
```

Comment:
this may get really annoying if there's little traffic and the interval is quite low.
how about trace maybe?

Reply:
i mean, it isn't classic "tracing" use of trace, but I guess it fits the use case?

Reply:
it's by default 1s, I think it is good for debug

Reply:
ok